languages/cpp/tree-sitter/Parse_cpp_tree_sitter.ml

```diff
@@ -4982,6 +4989,38 @@ let parse file =
       | Left prog -> prog
       | Right _ -> failwith "not a program")
 
+(* See th enote in Dummy_preprocessor.mli *)
+(* FIXME: We run the dummy preprocessor on the read file, before it goes to the
```

Comment:
In that case we should open an issue to address this later? 
Or make that into a simple TODO (preferred perhaps).